Makefile

```diff
@@ -29,5 +29,5 @@ pkgs/npm/README.md src/README.md: README.md
 
 .PHONY: test-nix
 test-nix:
-	nix run .#defang-cli
+	nix run .#defang-cli --extra-experimental-features flakes --extra-experimental-features nix-command
```

Comment:
nix demanded that i add these flags when i ran my git hooks. I'm not sure what's different between my machine and yours @lionello or if others need this as well?